### PR TITLE
Temp dev

### PR DESCRIPTION
**Pull request by vishalg8454**
This notes adds these missing routes as pointed out in revision comments "-- Aside from main page, other pages are not working as expected. -- check routing issues, fix that and resubmit"

- `/labels`
- `/profile`

### CODE REVIEW

File: src/components/navigation-drawer/NavigationDrawer.jsx [hunk 1]
```diff
@@ -49,7 +48,11 @@ const NavigationDrawer = () => {
             <span className="navigation-item-text">Home</span>
           </li>
         </NavLink>
-        <NavLink to="/labels" className="navlink">
+        <NavLink
+          to="/labels"
+          className="navlink"
+          style={({ isActive }) => (isActive ? activeStyle : undefined)}
```
Comment (Shailseen) [inline src/components/navigation-drawer/NavigationDrawer.jsx:54]: You can give style={activeStyle} only .

**Review by Shailseen — APPROVED** [review]
All code looks fine.
In mobile version app active link style is not working and all component works fine.
All looks fine you can proceed to merge.

File: src/components/navigation-drawer/NavigationDrawer.jsx (hunk 1 above)
Comment (vishalg8454) [reply src/components/navigation-drawer/NavigationDrawer.jsx:54]: oh, okay

File: src/context/note-context.js [hunk 2]
```diff
@@ -235,19 +235,20 @@ const NoteProvider = ({ children }) => {
...
-    setNotesList(() => temp);
+    notesList.sort((a, b) => new Date(a.dateTime) - new Date(b.dateTime));
     navigate("/home");
   }
 
   function sortOldToNew() {
-    const temp = notesList;
-    temp.sort((a, b) => new Date(b.dateTime) - new Date(a.dateTime));
-    setNotesList(() => temp);
+    notesList.sort((a, b) => new Date(b.dateTime) - new Date(a.dateTime));
```
Comment (Developers-codz) [inline src/context/note-context.js]: you both functions `sortOldToNew` and `sortNewToOld` working opposite

File: src/routes/archive-page/ArchivePage.jsx [hunk 3]
```diff
@@ -1,9 +1,14 @@
 import "./archive-page.css";
 import { useNote } from "../../context/note-context";
+import { useUser } from "../../context/user-context";
 import { Note } from "../../components";
```
Comment (Developers-codz) [inline src/routes/archive-page/ArchivePage.jsx:4]: can use absolute path

**Review by Developers-codz — CHANGES_REQUESTED** [review]
Found one bug fix that else look good to go 
Also after refreshing the browser the app link broke to fix that you can create a `_redirects` file in public folder and add  `/*  /index.html  200`

File: src/context/note-context.js (hunk 2 above)
Comment (vishalg8454) [reply src/context/note-context.js]: good catch 😁, thanks

File: src/routes/archive-page/ArchivePage.jsx (hunk 3 above)
Comment (vishalg8454) [reply src/routes/archive-page/ArchivePage.jsx:4]: ah, yes

**Pull request merged**